src/ZfcDatagrid/Renderer/JqGrid/View/Helper/Columns.php

```diff
@@ -36,10 +36,8 @@ private function translate($message)
 
         if (null === $this->translator) {
             if ($this->getServiceLocator()
-                ->getServiceLocator()
```

Comment:
the first getServiceLocator method return a [`Zend\ServiceManager\ServiceLocatorInterface`](https://github.com/zendframework/zend-servicemanager/blob/release-2.7.5/src/ServiceLocatorInterface.php#L15), there is no getServiceLocator method, only a get and has-method. thats why i removed the second call.

the same workflow is also in [ZfcDatagrid\Renderer\BootstrapTable\View\Helper\TableRow](https://github.com/ThaDafinser/ZfcDatagrid/blob/master/src/ZfcDatagrid/Renderer/BootstrapTable/View/Helper/TableRow.php#L22)